LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/DefaultLoRaDataRequestHandler.cs

```diff
@@ -141,8 +146,6 @@ public async Task<LoRaDeviceRequestProcessResult> ProcessRequestAsync(LoRaReques
                     // In multi gateway scenarios it means the another gateway was faster than using, can stop now
                     if (fcntDown <= 0)
                     {
```

Comment:
Did you remove the logging because it is done somewhere else?

Reply:
Yes. This was a duplicate logging event.